modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -370,57 +379,127 @@ test.describe('Manage object definitions through Model Builder', () => {
+			status: {code: 1},
+			titleObjectFieldName: 'id',
+		});
 
-		await modelBuilderPage.leftSidebarItems
-			.filter({hasText: objectDefinition.name})
-			.click();
+		objectDefinitions.push(department);
 
-		await expect(
-			modelBuilderPage.objectDefinitionNodes.filter({
-				hasText: objectDefinition.name,
-			})
-		).toBeVisible();
+		const employee = await apiHelpers.objectAdmin.postObjectDefinition({
+			active: false,
+			label: {
+				en_US: 'Employee',
+				pt_BR: 'Funcionario',
+			},
+			name: 'Employee',
+			objectFolderExternalReferenceCode:
+				objectFolder.externalReferenceCode,
+			panelCategoryKey: 'site_administration.design',
+			pluralLabel: {
+				en_US: 'Employees',
+				pt_BR: 'Funcionarios',
+			},
+			scope: 'site',
+			status: {code: 1},
+			titleObjectFieldName: 'name',
+		});
 
-		await expect(
-			modelBuilderPage.rightSidebar.getByTitle(
-				objectDefinition.name + ' Details'
-			)
-		).toBeVisible();
+		objectDefinitions.push(employee);
 
-		const details = [
-			'Label',
-			'Plural Label',
-			'Table Name',
-			'Activate Object',
-			'Entry Title Field',
-			'Scope',
-			'Panel Link',
-		];
+		await modelBuilderPage.goto({objectFolderName: objectFolder.name});
 
-		for (let i = 0; i < details.length; i++) {
-			const detail = details[i];
+		for (const objectDefinition of [department, employee]) {
+			await modelBuilderPage.leftSidebarItems
+				.filter({hasText: objectDefinition.label['en_US']})
+				.click();
 
 			await expect(
-				modelBuilderPage.rightSidebar
-					.filter({hasText: detail})
-					.filter({hasText: objectDefinition.name})
+				modelBuilderPage.rightSidebar.getByTitle(
+					`${objectDefinition.label['en_US']} Details`
+				)
 			).toBeVisible();
+
+			// Object Data Container
+
+			await expect(
+				modelBuilderPage.rightSidebarObjectDefinitionLabel
+			).toHaveValue(objectDefinition.label['en_US']);
+
+			await modelBuilderPage.rightSidebarObjectDefinitionLabelLocalizationButton.click();
+
+			await page
+				.getByRole('menuitem', {name: 'pt_BR Translated'})
+				.click();
+
+			await expect(
+				modelBuilderPage.rightSidebarObjectDefinitionLabel
+			).toHaveValue(objectDefinition.label['pt_BR']);
+
+			await page.keyboard.press('Escape');
+
+			await expect(
+				modelBuilderPage.rightSidebarObjectDefinitionPluralLabel
+			).toHaveValue(objectDefinition.pluralLabel['pt_BR']);
+
+			await modelBuilderPage.rightSidebarObjectDefinitionPluralLabelLocalizationButton.click();
+
+			await page.getByRole('menuitem', {name: 'en_US Default'}).click();
+
+			await expect(
+				modelBuilderPage.rightSidebarObjectDefinitionPluralLabel
+			).toHaveValue(objectDefinition.pluralLabel['en_US']);
+
+			await page.keyboard.press('Escape');
+
+			objectDefinition.active
+				? await expect(
+						modelBuilderPage.rightSidebarObjectDefinitionActivateObject
+					).toBeChecked()
+				: await expect(
+						modelBuilderPage.rightSidebarObjectDefinitionActivateObject
+					).not.toBeChecked();
```

Comment:
i'm not so sure why you did this conditional here, isn't `objectDefinition.active` controlled by us? so it couldn't change on its own and we would know exactly what to test when reaching this point of the test. could you please explain to me?

Reply:
Hello @rebsilva, we can define the active property in the object definition post. But even setting the property to true had no effect, because I can only activate published object definitions. 

So I needed to add an object field and change the status to get the active object definition.

And about the ternary operation, I managed to find an elegant way to do this:

```
await expect(
    modelBuilderPage.rightSidebarObjectDefinitionActivateObject
).toBeChecked({checked: objectDefinition.active});

```

see all changes [here](https://github.com/liferay-objects/liferay-portal/compare/99ea8ee8d1272130833a52168bfb3621d28233c0..ac2767ca0db6bf966ad824a7812e2c7c98ee10b4)

Reply:
i see, thanks for explaining! the change seems great!